### PR TITLE
Sync main with develop 

### PR DESCRIPTION
**Pull request by Pavangbhat**

### CODE REVIEW

File: src/profile/utils.js [hunk 1]
```diff
@@ -104,13 +141,15 @@ function createCard({ oldData, newData, userId, username, profileDiffId }) {
...
+  const diffValues = checkDifferentValues(oldData, newData);
+
   const oldUserInfoList = createCardComponent({
     tagName: 'ul',
     className: 'userInfoListContainer',
     parent: oldDataContainer,
   });
 
-  displayList(oldData, oldUserInfoList);
+  displayList(oldData, oldUserInfoList, diffValues, OLD_DATA);
```
Comment (rajakvk) [inline src/profile/utils.js:152]: NIT: Pass parameters in an object like displayList(options);

File: src/profile/utils.js [hunk 2]
```diff
@@ -130,7 +169,7 @@ function createCard({ oldData, newData, userId, username, profileDiffId }) {
     parent: newDataContainer,
   });
 
-  displayList(newData, newUserInfoList);
+  displayList(newData, newUserInfoList, diffValues, NEW_DATA);
```
Comment (rajakvk) [inline src/profile/utils.js:172]: NIT: Pass parameters in an object like displayList(options);

File: src/profile/utils.js (hunk 1 above)
Comment (Pavangbhat) [reply src/profile/utils.js:152]: Thank you @rajakvk 
@vin18 let's consider while refactoring

Comment (Pavangbhat) [reply src/profile/utils.js:152]: cc @rohan09-raj

**Review by rohan09-raj — APPROVED** [review]

File: src/profile/utils.js (hunk 1 above)
Comment (vin18) [reply src/profile/utils.js:152]: Got it, @Pavangbhat. Will handle this in global refactor PR. 
Thank you @rajakvk for the review.

**Pull request merged**